lib/server.go

```diff
@@ -2338,11 +2500,11 @@ func (srv *Server) _startConsensus() {
 		}
 
 		select {
-		// case consensusEvent := <-srv.fastHotStuffEventLoop.GetEvents():
-		// 	{
-		// 		glog.Infof("Server._startConsensus: Received consensus event for block height: %v", consensusEvent.TipBlockHeight)
-		// 		srv._handleFastHostStuffConsensusEvent(consensusEvent)
-		// 	}
+		case consensusEvent := <-srv.fastHotStuffConsensus.fastHotStuffEventLoop.GetEvents():
```

Comment:
Huh. That's all we had to add eh? Pretty elegant...

Reply:
Yup. That's all we need :)